src/views/LeadAdd/LeadProcessor/index.js

```diff
@@ -178,6 +208,7 @@ function LeadProcessor(props) {
                     error: false,
                     pristine: isDefined(serverId),
                 },
+                leadState: LEAD_STATUS.pending,
```

Comment:
Why set it to pending? The word pending is a bit misleading.
The state is pending. But, LEAD_STATUS.pending is actually working.
We should only set it to pending when the request is being made!